src/server/utilities/liteATITracking/viewTracking/index.ts

```diff
@@ -0,0 +1,20 @@
+export default () => {
+  const options = { threshold: 0.5 };
+  const observer = new IntersectionObserver(entries => {
+    console.log('INTERSECTION OBSERVER LOADED');
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
+        console.log('IS INTERSECTING');
+        const { target } = entry;
+        const atiURL = target.getAttribute('data-lite-ati-view-tracking');
```

Comment:
let's use `LITE_ATI_VIEW_TRACKING` from src/app/hooks/useViewTracker/index.jsx

```suggestion
        const atiURL = target.getAttribute(LITE_ATI_VIEW_TRACKING);
```

Reply:
Hello, I previously tried this with the clickTracking too, but I get this error: `Uncaught ReferenceError: _app_hooks_useViewTracker__WEBPACK_IMPORTED_MODULE_2__ is not defined`.

I think it's partly due to the function being serialised.

Reply:
Ah OK, I didn't realise this.